docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 45
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Mimir through the Prometheus data source type.
+
+## Before you begin
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. In the **HTTP** > **Prometheus server URL** field, enter a server URL:
+
+   - If you deployed Mimir via the `mimir-distributed` Helm chart,
+     the default URL inside the cluster is `http://<HELM-RELEASE>-nginx.<MIMIR-NAMESPACE>.svc/prometheus`,
+     and `http://<INGRESS-HOST>/prometheus` from outside, provided that you set up an ingress.
+
+   - If you deployed Mimir via Jsonnet and Tanka,
+     use `http://localhost:<MIMIR-PORT>`.
+
+   - If you are running microservices, point to the proxy in front of Mimir
+     (such as Nginx or the GEM gateway) rather than to one component of Mimir.
+
+## Steps
+
+1. To view metrics in Grafana, select **Explore**.
```

Comment:
After going to Explore, you also need to select the newly created data source at the top left.